autoscheduler/frontend/src/.eslintrc.js

```diff
@@ -46,13 +46,23 @@ module.exports = {
     "react/jsx-filename-extension": [1, {"extensions": [".jsx", ".tsx"]}],
     "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.ts", "**/*.test.tsx", "**/*.config.js"]}],
     "lines-between-class-members": ["error", "always", { exceptAfterSingleLine: true }],
+    "react/prop-types": 0,
+    "no-plusplus": ["error", {allowForLoopAfterthoughts: true}],
+    "no-mixed-operators": ["error", {
```

Comment:
I went ahead and made the linting rule that y'all suggested. This should warn about comparison operators but not arithmetic operators.